docs/02-admin/01-installation/docker.md

```diff
@@ -1,26 +1,33 @@
 # Docker Installation
 
 > [!NOTE]
-> Docker installation is currently not advised for production use. Try the [Bare Metal installation](./bare_metal.md) instead.
+> Docker installation is currently not advised for production use. Try the [Bare Metal installation](./bare_metal.md)
+> instead.
+
+> [!IMPORTANT]  
+> If you were already using docker in production, please see the [migration guide](#migration-guide) for updates
 
 ## System Requirements
 
 - Docker Engine
 - Docker Compose V2
 
-  > If you are using Compose V1, replace `docker compose` with `docker-compose` in those commands below.
+> [!WARNING]  
+> `docker-compose` is **not** supported!
```

Comment:
This can be very confusing I believe. "Docker compose" is supported and even the `docker-compose` command is supported, if you download the [standalone docker compose](https://docs.docker.com/compose/install/standalone/)). So please revert this sentence back.

Reply:
Reverted in [0a8a3ad](https://github.com/MbinOrg/mbin/pull/1021/commits/0a8a3adec9edd4f6772cd9121bfd5922bec64ee2)